### PR TITLE
Introduce kebechet metrics, some refactoring and update README

### PR DESCRIPTION
**Pull request by pacospace**
Fixes: https://github.com/thoth-station/metrics-exporter/issues/387

## This introduces a breaking change

- [ ] Yes
- [x] No

## This Pull Request implements

- Add Argo workflow metrics for kebechet workflows.
- Refactor some classes introducing Configuration file.
- Introduce docs about adding metrics for Argo workflows for a certain component and for Ceph for a new ResultStore.

### CODE REVIEW

**Comment by goern** [comment]
Isn’t coala superseded by pre-commit?! @harshad16

File: thoth/metrics_exporter/configuration.py [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+    # Workflows middletier namespace
+    _WORKFLOW_CONTROLLER_INSTANCE_MIDDLETIER_NAMESPACE = os.environ["WORKFLOW_METRICS_MIDDLETIER_PROMETHEUS_INSTANCE"]
+    _THOTH_MIDDLETIER_NAMESPACE = os.environ["THOTH_MIDDLETIER_NAMESPACE"]
+
+    # Workflows amun-inspection namespace
+    _WORKFLOW_CONTROLLER_INSTANCE_AMUN_INSPECTION_NAMESPACE = os.environ[
+        "WORKFLOW_METRICS_AMUN_INSPECTION_PROMETHEUS_INSTANCE"
+    ]
+    _THOTH_AMUN_INSPECTION_NAMESPACE = os.environ["THOTH_AMUN_INSPECTION_NAMESPACE"]
```
Comment (fridex) [inline thoth/metrics_exporter/configuration.py:47]: These do not need to be private, they are used outside of the class.

Comment (pacospace) [reply thoth/metrics_exporter/configuration.py]: Updated :) Thanks!!

Comment (fridex) [reply thoth/metrics_exporter/configuration.py]: The README file deserves some changes as well :)

Comment (pacospace) [reply thoth/metrics_exporter/configuration.py]: Done :D Thanks :)

**Pull request merged**